Docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+services: 
+  app:
+    build: .
+    ports:
+      - "5000:5000"
+    command: bundle exec rails s -p 5000
+    depends_on:
+      - iis-sandbox
+      - v2-to-fhir
+    links:
+      - iis-sandbox
+      - v2-to-fhir
+    environment:
+      IIS_SANDBOX_USERNAME: mitre
+      IIS_SANDBOX_PASSWORD: mitre
+      IIS_SANDBOX_FACILITY_ID: MITRE Healthcare
+      IIS_SANDBOX_HOST: http://localhost:8081
+      V2_TO_FHIR_HOST: http://localhost:3000
```

Comment:
Do these need to be set if we already have these same defaults in `.env.production`?

I would do something similar to what we have in [the docker-compose for health_cards](https://github.com/dvci/health_cards/blob/nginx/docker-compose.yml#L8-L10) where the docker-compose is configured to pull from the system environment, but we provide a [`.env` with default values](https://github.com/dvci/health_cards/blob/nginx/.env) 

[Docker docs on docker-compose envs with .env default](https://docs.docker.com/compose/environment-variables/#substitute-environment-variables-in-compose-files)